app/javascript/dashboard/routes/dashboard/helpcenter/components/ArticleTable.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="w-full">
     <div
-      class="my-0 py-0 px-4 grid grid-cols-6 md:grid-cols-7 lg:grid-cols-8 gap-4 border-b border-slate-100 dark:border-slate-700 sticky top-16 bg-white dark:bg-slate-900"
+      class="my-0 py-0 px-4 grid grid-cols-6 z-10 md:grid-cols-7 lg:grid-cols-8 gap-4 border-b border-slate-100 dark:border-slate-700 sticky top-16 bg-white dark:bg-slate-900"
```

Comment:
These changes will fix the overflow issue

**Before**
![Screenshot 2024-01-24 at 9 31 56 AM](https://github.com/chatwoot/chatwoot/assets/64252451/68a8cfec-631d-458b-be2b-411c68ced560)

**After**
<img width="210" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/9384c56a-eea2-4b67-aede-5f5a256ae161">